pwa/src/services/getConfig.ts

```diff
@@ -0,0 +1,8 @@
+import Epe from "./../../static/configFiles/epe.json";
+
+export const getConfig = (domain: string): Record<string, any> | undefined => {
+  switch (domain) {
+    case "open.epe.nl":
```

Comment:
You can test this by adding a case "localhost" to this.

Reply:
Dont forget to set `GATSBY_ENV_VARS_SET` to `false`

Reply:
Hmm. I don't think it should be `false`. It's a bit odd. The `.env` doesn't actually "exist" when this JSON-config exists. But in our code base the exist simultaneously.

Basically there are two scenario's: 

- The .env is set, and in it, the `GATSBY_ENV_VARS_SET` is set to `"true"`. In this case the JSON-config files are _not_ used.
- The .env is NOT set (it doesn't exist, because the implementing server didn't create it), and therefore the `GATSBY_ENV_VARS_SET` is non-existent (so not `=== "true"`). In this case the JSON-config is used.